_kbarticles/2024-12-10-using-opensearch-dashboards.md

```diff
@@ -0,0 +1,71 @@
+
+![Screenshot of Dashboard interface showing a query for router logs for a given CloudFoundry space and application]({{ site.baseurl }}/assets/images/content/dashboard-router.png)
+
+The next step is to visualize your search results based on a specific field. To visualize request logs over time, choose the `@timestamp` field from the left sidebar of "Available fields". Then, click "Visualize".
+
+![Screenshot of Dashboards interface showing the selection of @timestamp as field to use for visualizing request logs]({{ site.baseurl }}/assets/images/content/opensearch-timestamp.png)
+
+By default, visualizing logs based on `@timestamp` will produce a histogram chart. To change the chart type to line, which might be more useful for this type of data, click the "Metrics & axes" link in the chart configuration panel on the right side of the screen. Then, under "Metrics" and "Count", select "Line" from the "Chart type" drop-down. Finally, click the "Update" button at the bottom right of the screen and the chart should update to a line chart.
+
+![Screenshot of Dashboards visualization interface showing a line chart of requests over time with a panel for chart configuration options visible on the right side of the screen. In the chart configuration options, "line" is selected as the chart type]({{ site.baseurl }}/assets/images/content/dashboard-line.png)
+
+## Fields for router requests
+
+Listed below are the explanations of some field names for router (RTR) logs:
+
+- **rtr.app.id**: The application guid
+- **rtr.hostname**: The domain/hostname the request was sent to (e.g test.app.cloud.gov)
+- **rtr.http_user_agent**: What user agent the request came from (Chrome, Firefox, Curl, etc…)
+- **rtr.path**: The specific url path that was requested (e.g. /my/test/page)
+- **rtr.status**: Gives the status of the request (200, 404, etc…)
+- **rtr.verb**: The type of request (POST, GET, etc...)
+- **rtr.x_forwarded_for**: The IP address the request came from
+- **rtr.timestamp**: The time of the request in UTC
+
+The full list of fields available for router logs can be found in [our Opensearch field mapping configuration](https://github.com/cloud-gov/opensearch-boshrelease/blob/5100229e391237f4a41df023338a247c55276ee8/jobs/opensearch_templates/templates/component-index-mappings-app.json.erb#L44).
+
+## Generate report of logs.
+
+Opensearch allows for [generating a report](https://opensearch.org/docs/latest/reporting/report-dashboard-index/) from a saved search. This is done by choosing a search and clicking the **Reporting** menu option.
+
+![Screenshot of Dashboards interface showing the selection of ]({{ site.baseurl }}/assets/images/content/opensearch-reporting.png)
+
+> Reports have a non-configurable 10,000 row limit. They have no explicit size limit (for example, MB),
```

Comment:
The formatting on block quotes doesn't look great. Maybe we remove the `>` and just prepend this section with `Please note:` in bold